hack/cherry-pick.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+# See HACKING.md for usage
+source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
+
+repo="${UPSTREAM_REPO:-k8s.io/kubernetes}"
+package="${UPSTREAM_PACKAGE:-pkg/api}"
+UPSTREAM_REPO_LOCATION="${UPSTREAM_REPO_LOCATION:-../../../${repo}}"
+pr="$1"
+
+if [[ "$#" -ne 1 ]]; then
+  echo "You must supply a pull request by number or a Git range in the upstream ${repo} project" 1>&2
+  exit 1
+fi
+os::build::require_clean_tree # Origin tree must be clean
+
+patch="${TMPDIR:-/tmp}/patch"
+rm -rf "${patch}"
+mkdir -p "${patch}"
+patch="${patch}/cherry-pick"
+
+if [[ ! -d "${UPSTREAM_REPO_LOCATION}" ]]; then
+  echo "Expected ${UPSTREAM_REPO_LOCATION} to exist" 1>&2
+  exit 1
+fi
+
+lastrev="${NO_REBASE-}"
+if [[ -z "${NO_REBASE-}" ]]; then
+  lastrev="$(go run ${OS_ROOT}/tools/godepversion/godepversion.go ${OS_ROOT}/Godeps/Godeps.json ${repo}/${package})"
```

Comment:
Dependencies on godep should be removed.  I don't see us using long term and we don't have any of the broken out pieces (api, client-go, image-registry) using it.  They're all using glide.

Reply:
I can remove cherry-pick if you don't think you'll need it.